packages-native/intro-screen/src/SwipeableContainer.tsx

```diff
@@ -0,0 +1,310 @@
+import { useState, createElement, ReactNode, useCallback, ComponentClass, Fragment } from "react";
+import {
+    StyleSheet,
+    FlatList,
+    View,
+    Dimensions,
+    Text,
+    TouchableOpacity,
+    Platform,
+    I18nManager,
+    ViewStyle,
+    NativeSyntheticEvent,
+    LayoutChangeEvent,
+    TouchableNativeFeedback
+} from "react-native";
+import DeviceInfo from "react-native-device-info";
+import { IntroScreenStyle } from "./ui/Styles";
+
+const isIphoneWithNotch = Platform.OS === "ios" && DeviceInfo.hasNotch();
+
+const isAndroidRTL = I18nManager.isRTL && Platform.OS === "android";
+
+interface SwipeableContainerProps {
+    skipLabel: string;
+    doneLabel: string;
+    nextLabel: string;
+    prevLabel: string;
+    showDoneButton: boolean;
+    showSkipButton: boolean;
+    showNextButton: boolean;
+    showPrevButton: boolean;
+    onSlideChange: (next: number, previous: number) => void;
+    bottomButton: boolean;
+    numberOfButtons: number;
+    renderDoneButton: () => ReactNode;
+    renderSkipButton: () => ReactNode;
+    renderPrevButton: () => ReactNode;
+    renderNextButton: () => ReactNode;
+    onDone: () => void;
+    onSkip: () => void;
+    slides: [];
+    hidePagination: boolean;
+    styles: IntroScreenStyle;
+    showMode: string;
+}
+
+const Touchable: ComponentClass<any> = Platform.OS === "android" ? TouchableNativeFeedback : TouchableOpacity;
+
+export const SwipeableContainer = (props: SwipeableContainerProps): JSX.Element => {
+    const dimensions = Dimensions.get("window");
+    const [width, setWidth] = useState(dimensions.width);
+    const [activeIndex, setActiveIndex] = useState(0);
+    const [flatList, setFlatList] = useState();
+
+    const goToSlide = useCallback(
+        (pageNum: number) => {
+            setActiveIndex(pageNum);
+            if (flatList) {
+                flatList.scrollToOffset({
+                    offset: _rtlSafeIndex(pageNum) * width
+                });
+            }
+        },
+        [width, flatList]
+    );
+
+    const _onNextPress = (): void => {
```

Comment:
No need to start with underscore  for any variables
```suggestion
    const onNextPress = (): void => {
```

Reply:
Done